about.html

```diff
@@ -23,8 +23,20 @@
         <li class="menu-highlight"><a href="#">Stoa Campaign</a></li>
       </ul>
     </navbar>
-    <!-- Menu Mobile-->
-    <navbar class="nav"><img src="images/mobile_menu.png" alt="hamburger"></navbar>
+    <nav>
+      <a class="hamburger">
+        <img src="images/mobile_menu.png" alt="hamburger">
+      </a>
+    </nav>
+    <nav>
+
+      <ul class="mobile-menu">
+        <a class="close"><img src="images/cancel.png" alt=""></a>
+        <li class="menu-link"><a href="index.html">Home</a></li>
+        <li class="menu-link"><a href="about.html">About</a></li>
+        <li class="menu-link menu-highlight"><a href="#">Stoa</a></li>
+      </ul>
+    </nav>
   </header>
 
 
```

Comment:
- [ ] Kindly make sure to add a background to the Main section so it looks more clean and stylish
- [ ] Kindly consider adding a border to the paragraph in the main section so it looks exactly like the CC design

|Your page|CC design|
|-|-|
|![Screenshot_20230318_001201](https://user-images.githubusercontent.com/25227022/226067433-08e1d8dc-7850-4f67-b60b-616275fe7bfd.png)|![Screenshot_20230318_001118](https://user-images.githubusercontent.com/25227022/226067442-d7378eab-0421-4496-812e-e401cee423a3.png)|